tests/mpi/hp_integrate_difference.cc

```diff
@@ -0,0 +1,254 @@
+// ---------------------------------------------------------------------
+//
+// Copyright (C) 2009 - 2017 by the deal.II authors
```

Comment:
So... how much do we care about these dates?

Reply:
I think this is actually correct. I copied another test (for the non-hp case) and made minor modifications. So the previous copyright years remain correct. I could have added 2018, but that will automatically happen once we run the script on this -- so I didn't bother.